### PR TITLE
Remove Toxiin generation When dead

### PR DESCRIPTION
**Pull request by Bod9001**
### Changelog:

CL: [Balance] Remove Toxiin generation When dead
CL: [fix] Fixes face huggers not implanting eggs Properly
CL: [fix] Fixes Suit fabricators not Working

### CODE REVIEW

File: UnityProject/Assets/Scripts/HealthV2/Living/PolymorphicSystems/NaturalChemicalReleaseSystem.cs [hunk 1]
```diff
@@ -134,7 +134,7 @@ public void ToxinGeneration(float HeartEfficiency)
 			float Multiplier = HeartEfficiency;
 			if (HeartEfficiency == 0)
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/HealthV2/Living/PolymorphicSystems/NaturalChemicalReleaseSystem.cs:135]: this doesn't say "dead" specifically. Is there any situation where you have a bad heart but are still alive?

Comment (Bod9001) [reply UnityProject/Assets/Scripts/HealthV2/Living/PolymorphicSystems/NaturalChemicalReleaseSystem.cs:135]: You can be alive with no heart but None of the systems work such as Oxygenation

**Comment by Bod9001** [comment]
Scene changes are to add the Suit fabricators to APC Since they powered now

File: UnityProject/Assets/Scripts/HealthV2/Living/PolymorphicSystems/NaturalChemicalReleaseSystem.cs (hunk 1 above)
Comment (corp-0) [reply UnityProject/Assets/Scripts/HealthV2/Living/PolymorphicSystems/NaturalChemicalReleaseSystem.cs:135]: would it be better to be more specific and check if the creature is dead instead?

**Pull request merged**